homeassistant/components/zha/number.py

```diff
@@ -949,3 +949,58 @@ class AqaraThermostatAwayTemp(
     _attr_mode: NumberMode = NumberMode.SLIDER
     _attr_native_unit_of_measurement: str = UnitOfTemperature.CELSIUS
     _attr_icon: str = ICONS[0]
+
+
+@CONFIG_DIAGNOSTIC_MATCH(
+    cluster_handler_names="tuya_manufacturer",
+    quirk_classes={"zhaquirks.tuya.ts0601_trv.ZonnsmartTV01_ZG"},
+)
+# pylint: disable-next=hass-invalid-inheritance # needs fixing
+class ZonnSmartTemperatureCalibration(
+    ZHANumberConfigurationEntity, id_suffix="temperature_calibration"
+):
+    """ZonnsSmart temperature offset configuration entity."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_native_min_value: float = -5
+    _attr_native_max_value: float = 5
+    _attr_native_step: float = 0.5
+    _zcl_attribute: str = "temperature_calibration"
+    _attr_name: str = "Temperature calibration offset"
+    _attr_icon: str = ICONS[0]
+
+    async def async_set_native_value(self, value: float) -> None:
+        """Set the offset value."""
+        await super().async_set_native_value(value * 10.0)
```

Comment:
I think you can set `_attr_multiplier` to `0.1` and then remove this.
(It's named "multiplier" because it goes from ZCl -> ZHA IIRC)

Reply:
Thanks ! That simplifies the code a lot and get rid of the tests. I didn't see that multiplier.